morebits.js

```diff
@@ -2104,6 +1921,264 @@ Morebits.wiki.page = function(pageName, currentAction) {
 		}
 	};
 
+	/**
+	 * returns a string containing the name of the loaded page, including the namespace
+	 */
+	this.getPageName = function() {
+		return ctx.pageName;
+	};
+
+	/**
+	 * returns a string containing the text of the page after a successful load()
+	 */
+	this.getPageText = function() {
+		return ctx.pageText;
+	};
+
+	/**
+	 * `pageText` - string containing the updated page text that will be saved when save() is called
```

Comment:
Because of the odd behaviour it causes (in my editor, VScode, at least): putting @param makes the tooltip unnecessarily carry the parameter's description twice. Once at the top: just below the function prototype and once again in the parameters' section. This is especially poor-looking for those large param descriptions like in `this.setMinorEdit()`.

Make it jsdocs if you feel like it.